src/plugins/expressions/common/ast/format_expression.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ExpressionAstExpression } from './types';
+import { format } from './format';
+
+/**
+ * Given expression pipeline AST, returns formatted string.
+ *
+ * @param ast Expression pipeline AST.
+ */
+export function formatExpression(ast: ExpressionAstExpression): string {
+  return format(ast, 'expression');
+}
```

Comment:
split this into a separate file for consistency with how we split `parse_expression` into a separate file